bowling-ball/Frames.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingBall
+{
+    class Frames : IFrames
+    {
+        int[] EachFrame = new int[2];
+        const int LowestPossibleValue = 0;
+        const int HighestPossibleValue = 10;
+        const int EmptyFrame = -1;
+
+        public Frames()
+        {
+            this.EachFrame[0] = -1;
+            this.EachFrame[1] = -1;
+        }
+
+        public void SetFrame(int pin, int indexStatus) => this.EachFrame[indexStatus] = pin;
+
+        public int GetFrame(int indexStatus) => this.EachFrame[indexStatus];
+
+        public int CalculateScore()
+        {
+            return EachFrame[0] + EachFrame[1];
+        }
+
+        public bool IsSpare()

```

Comment:
repeated comments: why are so many methods `public`?
Access modifiers are made for a purpose. Please use them carefully. Not all methods need to be `public`.

Reply:
Because, I am trying to use methods from another class Frame in BowlingBall Class.